changelog/1576.md

```diff
@@ -0,0 +1,5 @@
+level: major
```

Comment:
Thanks for writing a changelog!

This patch seems like a big deal, but it's actually removing something that doesn't work, so it's not strictly a breaking change in the sense of semantic versioning.  It's still a pretty big deal, though :)

```suggestion
level: minor
```